core/src/com/softwarearchitecture/ECS/Components/MoneyComponent.java

```diff
@@ -0,0 +1,9 @@
+package com.softwarearchitecture.ecs.components;
+
+public class MoneyComponent {
+    public int balance;
+
+    public MoneyComponent(int balance) {
+        this.balance = balance;
+    }
```

Comment:
Could we rename balance to amount? This lets us reuse it for f.ex. how much money enemies drop as well as how much money the player has